Hello Microverse project

Gdiazdiaz opened the pull request:
### Hello Microverse project
In this pull request, the following was done:
- [x] Add html file with a hello world text
- [x] Add a css file
- [x] Create a test.md file and added it into a `.gitignore` file with `node_module`.
 
 The author of these changes is @Gdiazdiaz

Gdiazdiaz committed:
create a html file

create a html file with the words "Hello world"

Gdiazdiaz committed:
Add a css file

added a css file with color blue for the text

Gdiazdiaz committed:
Add class

Added a class of header to the header

Gdiazdiaz committed:
Created a md file

Created a md file that should be ignored

ridwanediallo reviewed (changes requested):
# Changes Required ♻️
## Hi @Gdiazdiaz , 
![AlmostThereReachGIF](https://user-images.githubusercontent.com/55918904/191301799-36d07166-0e35-4582-a231-7a2d780f7783.gif)

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


## Required Changes ♻️

- [ ]  Please I would recommend you to change the title of your pull request to something meaninful. For instance `Hello Microverse project`. And add a description to your pull request (The tasks that you've accomplished in the project). 
[See this video help you update you pull request](https://user-images.githubusercontent.com/89228743/197641077-e8d8dd6a-0720-471e-a0b3-e76e33d664a8.webm)

- [ ]  Please in your project create a `.gitignore` file and add into it your `test.md` and `node_module`.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Gdiazdiaz committed:
add md to ignore

Gdiazdiaz committed:
Update .gitignore

Gdiazdiaz committed:
Update .gitignore

Gdiazdiaz committed:
Edited .ignore file

edited the ignore file to ignore files with .md except the readme file, also added a command to ignore the node.module

ridwanediallo commented on .gitignore:
```diff
@@ -0,0 +1,3 @@
+*.md
+!README.md
+/node.module
```
- [ ] Please your .gitignore should contain these 👇️
```
test.md
node_module
```
Copy them and paste them into `.gitignore` file.

ridwanediallo reviewed (changes requested):
# Changes Required ♻️
## Hi @Gdiazdiaz , 
![AlmostThereReachGIF](https://user-images.githubusercontent.com/55918904/191301799-36d07166-0e35-4582-a231-7a2d780f7783.gif)

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


## Required Changes ♻️

- [ ]  Please you have not still added a description to your pull request. Watch demo video to where I typied `Add here the tasks you have accomplished` add this 
```
### Tasks accomplished:
 - Add html file with a hello world text
 - Add a css file
 - Create a test.md file and added it into a `.gitignore` file with `node_module`.
```
- [ ] It is important to add a description to your pull request because it helps the one who is going to look at you pull request to know what changes have been made in your project.

[See this video help you update you pull request](https://user-images.githubusercontent.com/89228743/197641077-e8d8dd6a-0720-471e-a0b3-e76e33d664a8.webm)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Gdiazdiaz committed:
Update gitignore to the correct comands

Gdiazdiaz committed:
Delete test.md

Gdiazdiaz committed:
Delete .gitignore

Gdiazdiaz committed:
Fix gitignore with correct comands

ridwanediallo reviewed (approved):
# STATUS: APPROVED 🟢
## Hi @Gdiazdiaz , 
![Congratulations!GIF](https://user-images.githubusercontent.com/55918904/191301390-43d62dfa-bfdc-49c5-943b-4d72f4ea5bb8.gif)

Well done on implementing the requested changes. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Gdiazdiaz committed:
Merge branch 'master' into Add-files
